frontend/src/components/events/sponsors/sponsor_gallery.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { Container, Col, Row } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import "./index.css";
+
+class SponsorGallery extends Component {
+    renderImage(imageUrl) {
+        return (
+            <Col className="logo-column" sm={3}>
+               <img 
+                src={imageUrl}
+                className="logo-picture"
+                alt="sponsor logo"
+            /> 
+            </Col>            
+        )
+    }
+    render() {
+        return (
+            <Container className="sponsor-logos">
+                <Row>
+                    {this.props.imageUrls.map(
+                        imageUrl => this.renderImage(imageUrl)
+                    )}
+                </Row>                
```

Comment:
Your use of bootstrap in the columns and rows really makes the logos look uniform and stylish